### PR TITLE
Add UCB exploration via. dropout

### PR DESCRIPTION
**Pull request by econti**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,9 +23,8 @@ This repo holds the open source machine learning code that powers [banditml.com]
 ## Supported models
 
 Models supported:
-- [x] Gradient boosted decision tree contextual bandit with ε-greedy exploration
```
Comment (econti) [inline README.md:26]: This will be a good chunk of work to implement, so holding off.

File: banditml_pkg/banditml/serving/predictor.py [hunk 2]
```diff
@@ -35,6 +35,11 @@ def __init__(
         # the ordered decisions that we need to score over.
         self.decisions = []
 
+        # if using dropout to get prediction uncertainty, how many times to score
+        # the same observation
+        self.num_times_to_score = 50
```
Comment (lgvital) [inline banditml_pkg/banditml/serving/predictor.py:40]: is 50 a general best practice / best guess? future hyper param?

Comment (econti) [reply banditml_pkg/banditml/serving/predictor.py]: yea, just a best guess, will need to be less for bigger models

Comment (econti) [reply banditml_pkg/banditml/serving/predictor.py]: the higher the better, but it will add latency to inference

**Pull request merged**